l1-contracts/contracts/common/L1ContractErrors.sol

```diff
@@ -251,6 +251,8 @@ error STMNotRegistered();
 error SystemLogsSizeTooBig();
 // 0x08753982
 error TimeNotReached(uint256 expectedTimestamp, uint256 actualTimestamp);
+//
```

Comment:
Yes, but tbh, having these selectors in comments is a bad practice. We can't verify it and generating those data takes time from developer, brr should remove it all together 

cc @koloz193 wdyt?

Reply:
fixed and make sense. 

if you decide to remove the comments at the end, I will fix the PR accordingly

Reply:
i think rather than removing it would be worth spending the time to get them to be autogenerated. A few times it's helped me figure out what error was being thrown in the server. Another option (not mutually exclusive) i talked with @StanislavBreadless about is having better translation in the server where we can look up an error based on the function selector. Overall I do think that its a value add even if it takes a small amount of time to generate/verify